cif_core.dic

```diff
@@ -28085,4 +28180,10 @@ save_
        _audit_support.special_details data items.
 
        Added opaque identifiers for diffrn_refln and refln categories.
+
+       Added _diffrn.flux_density, _diffrn.total_dose and
+       _diffrn.total_exposure_time as first step towards characterizing
+       radiation damage (bm).
+
+       Changed _type.purpose of _diffrn.total_dose to 'Measurand' (bm).
```

Comment:
```suggestion
```

Reply:
I would remove this line as the change to 'Measurand' was made in the bounds of this PR. That is, both the addition and all the changes to the `_diffrn.total_dose` data item fall under the previously sentence `Added _diffrn.flux_density, _diffrn.total_dose ...`.